pom.xml

```diff
@@ -234,7 +234,7 @@
                         <Implementation-Version>${spec.implementation.version}</Implementation-Version>
                         <Specification-Version>${spec.specification.version}</Specification-Version>
                         <Bundle-Description>
-                            Java(TM) Common Annotations ${spec.version} API Design Specification
+                            Java(TM) Jakarta Annotations ${spec.version} API Design Specification
                         </Bundle-Description>
```

Comment:
This should just be something like "Jakarta Annotations, Version ${spec.version}".
And obviously once the specification document is available it should be changed to match.